### PR TITLE
Add code-studio to import order rule

### PR DESCRIPTION
**Pull request by lfryemason**
In order to improve code cleanliness slightly, we are rolling out an eslint rule to the entire repo that ensure .js, .jsx, .ts and .tsx files have imports automatically sorted. 

We are rolling this out incrementally to prevent errors and decrease PR size. This PR removes a subset of folders from the rule exclusions.

FLUP of [#57247](https://github.com/code-dot-org/code-dot-org/pull/57247)

**For PR reviewers**: please double check that all comments in imports are appropriately moved with the line they are talking about.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/webpack.config.js [hunk 1]
```diff
@@ -80,6 +80,7 @@ const circularDependenciesSet = new Set(circularDependencies);
 // as we see our known circular dependencies, we're gonna remove them from our list. That way,
 // we can report at the end if any circular dependencies have been cleaned up.
 let seenCircles = new Set();
+let numUnresolvedCircles = 0;
```
Comment (lfryemason) [inline apps/webpack.config.js]: Adding some extra context to output to make debugging easier.

Comment (Hamms) [reply apps/webpack.config.js]: Would you mind extracting this change into its own PR? I like the new functionality, but I do want to keep it separate from the cleanup work that's supposed to have zero side effects. I'll also be interested to hear what @thomasoniii thinks about it!

File: apps/src/code-studio/pd/workshop_dashboard/components/survey_results/text_responses.jsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
+import he from 'he';
+import _ from 'lodash';
 import PropTypes from 'prop-types';
 import React from 'react';
 import {Well} from 'react-bootstrap'; // eslint-disable-line no-restricted-imports
-import _ from 'lodash';
-import he from 'he';
```
Comment (Hamms) [inline apps/src/code-studio/pd/workshop_dashboard/components/survey_results/text_responses.jsx:5]: Total aside, but: looks like we have a worthy challenger to the Go programming language in the "least-searchable name of all time" competition

**Review by Hamms — COMMENTED** [review]
Wooo, `code-studio`! Definitely feels like the most substantial of the remaining directories :tada: 

I'd ideally like to see them each in their own PR, but both the updates to circular dependency functionality and the import reorderings look good to me individually.

**Review by Hamms — APPROVED** [review]
LGTM; thank you!

**Pull request merged**